### ISSUE TITLE
Make BlitToDisplay use transparent shader.

### ISSUE THREAD

**Issue by art0007i** [Yellow-Dog-Man/Resonite-Issues#680]
### Is your feature request related to a problem? Please describe.

In OBS there is a "Allow Transparency" checkbox which works with unity (somehow) and using it it's possible to record transparent video.
Unfortunately the BlitToDisplay component renders the image with an opaque shader, which makes it so that the video is not transparent.

### Describe the solution you'd like

The BlitToDisplay component renders a transparent image with transparency.

### Describe alternatives you've considered

using this mod: https://github.com/art0007i/TransparentCameraEnabler
which simply changes the `TextureDisplayBlitter` class to use a "Unlit/Transparent" shader instead of the "Unlit/Texture" which it normally uses

**Comment by Frooxius** [comment]
Does this actually do what you expect it to? The blit is rendered on top of normal render, so I'd expect it show whatever is in the normal view below it.

**Comment by art0007i** [comment]
yeah it worked correctly. here's a youtube video demonstrating it https://www.youtube.com/watch?v=usowjujVKYA

**Comment by Frooxius** [comment]
Ok that seems good! Are there any downsides? Like does this produce any artifacts when rendering things? In the video I can see that things look a bit weird for a moment. I'm not sure if this should be a fixed change or if it should be a optional toggle.

**Comment by art0007i** [comment]
> In the video I can see that things look a bit weird for a moment.

If you mean the part where the game is still mirroring my headset, but it looks like it has a "ghost filter" on it, then that always happens when using the "allow transparency" setting in obs even without mods. I have no idea why that happens, but I'm guessing that the headset mirror that is displayed in the window has some unrelated data in the alpha channel and obs is capturing that.

I couldn't notice anything that looks off other than that. 

I think it should be a fixed change, because BlitToDisplay component already supports transparency, but not on the actual texture, but only on the background color. Here's two screenshots of what it looks like in obs without the mod and with it:

Without the mod:
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/19620451/9c970b95-571d-47a6-a7e4-5841e37696d8)

With the mod:
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/19620451/6c9265cc-f4ef-4f05-a3b2-075bc5fdb43c)

And this is the component setup I used in both screenshots:
![2023-11-15 21 47 21](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/19620451/828de695-8e68-4066-b602-3e48bdc0a5d8)

**Comment by Frooxius** [comment]
Merged this in 2026.5.19.1285! Thanks!

**Comment by lxw404** [comment]
This doesn't seem to be working in `Beta 2026.5.19.1285`.  Here is my OBS setup in OBS version `32.1.2`:

<img width="1760" height="873" alt="Image" src="https://github.com/user-attachments/assets/49fd50fb-7599-4571-b615-ccda0a36be92" />

Here's the Resonite `BlitToDisplay` setup as in the video above:

<img width="3840" height="2160" alt="Image" src="https://github.com/user-attachments/assets/877e2dc1-3fec-4c24-b68f-054c61b7f0b0" />

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/e3085d12-267e-4c1b-a9b3-a785c9afd996" />

The result is this:

<img width="2162" height="1518" alt="Image" src="https://github.com/user-attachments/assets/3c76393b-9456-4bff-908d-45591970ce0f" />

Not sure if I'm missing a step that wasn't shown in the video, or if this isn't actually fixed.

[BlitToDisplayTransparency - 2026.5.19.1285 - 2026-05-19 18_37_06.log](https://github.com/user-attachments/files/28037143/BlitToDisplayTransparency.-.2026.5.19.1285.-.2026-05-19.18_37_06.log)

**Comment by Frooxius** [comment]
Can you try grabbing the latest build of this and replacing it and see if that fixes it? 

https://github.com/Yellow-Dog-Man/Renderite

It's possible it didn't make it into the build.

**Comment by NepuShiro** [comment]
> Can you try grabbing the latest build of this and replacing it and see if that fixes it?
> 
> [Yellow-Dog-Man/Renderite](https://github.com/Yellow-Dog-Man/Renderite)
> 
> It's possible it didn't make it into the build.

It seems the wrong version of Renderite.Unity was pushed
as looking at the decomp, there's stuff related to jvyden's il2cpp renderer and a bunch of IsWine checks removed
And this fix isn't included

**Comment by Frooxius** [comment]
Chrum... something wrong got pulled in. Thanks, I'll sort it out!

**Comment by Frooxius** [comment]
Ok it should be sorted out now in 2026.5.20.222! Let me know if there' still issues!

**Comment by lxw404** [comment]
It seems that `2026.5.20.222` launches into a blank black screen and never continues loading.

I've verified the integrity of files in Steam after experiencing this and it doesn't change.

[2026.5.20.222 - Blank - Bootstrapper.log](https://github.com/user-attachments/files/28040136/2026.5.20.222.-.Blank.-.Bootstrapper.log)

[Blank - 2026.5.20.222 - 2026-05-19 21_16_43.log](https://github.com/user-attachments/files/28040138/Blank.-.2026.5.20.222.-.2026-05-19.21_16_43.log)

[2026.5.20.222 - Blank - Player.log](https://github.com/user-attachments/files/28040160/2026.5.20.222.-.Blank.-.Player.log)
